src/journey-maps/angular/components/template-renderer/template-renderer.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  Input,
+  OnChanges,
+  SimpleChanges,
+  TemplateRef,
+  ViewChild,
+} from '@angular/core';
+
+import { SbbTemplateType } from '../../journey-maps.interfaces';
+
+@Component({
```

Comment:
At first glance a structural directive might be a better match for this functionality.
(e.g. maybe extend https://github.com/angular/angular/blob/13.3.3/packages/common/src/directives/ng_template_outlet.ts#L10-L71 )
Or is there a specific reason why you prefer a wrapper component?

Reply:
> Or is there a specific reason why you prefer a wrapper component?
No - done.